Minor fixes - update to OS/platform and services matrix and App Center bug tracker integration and four other articles

mahab339 opened the pull request:
- In deploy Azure DevOps builds with App Center article, change (... for querying for the Store IDs using .) and (... to create a new connection.Paste the API key you copied...) to (... for querying for the Store IDs.) and (... to create a new connection. Paste...) respectively.

- In Intune company portal distribution article, change (... global admin of your company's Azure Active Directory global admin will need to go to... ) to (... Azure Active Directory will need to go to... ).

- In publish an iOS package to the App Store (Production) article, change (https://developer.apple.com/support/app-review/) to (https://developer.apple.com/app-store/review/). This fixes Apple review article broken URL.

- In OS/platform and services matrix article:
a - Add .NET MAUI Android and .NET MAUI iOS and their supported services in App Center: distribute, diagnostics, and analytics. 
References:
     - [App Center Distribute – MAUI and Xamarin In-app updates](https://learn.microsoft.com/en-us/appcenter/sdk/distribute/xamarin).
     - [App Center Crashes (MAUI and Xamarin)](https://learn.microsoft.com/en-us/appcenter/sdk/crashes/xamarin).
     - [App Center Analytics (MAUI and Xamarin)](https://learn.microsoft.com/en-us/appcenter/sdk/analytics/xamarin).
At least I am sure of diagnostics and analytics because I've used them. 

  b - Change (* - For UWP, HoloLens support hasn't been verified.) to (\\* - For UWP...) for literal render of (*).

- In App Center bug tracker integration article, change (\[work item types API](/rest/api/vsts/wit/work%20item%20types) for additional information.) to (... vsts/wit/) for additional...) as former one displays 404 page.

- In App Center webhooks article, change (For Distribute, the webhook will be triggered when you when a new version is released to a distribution group.) to (... will be triggered when a new version is released...).

mahab339 committed:
Information update - update to OS/platform and services matrix article

- Add .NET MAUI Android and .NET MAUI Android and their supported services in App Center: distribute, diagnostics, and analytics.
References:
-- https://learn.microsoft.com/en-us/appcenter/sdk/analytics/xamarin
-- https://learn.microsoft.com/en-us/appcenter/sdk/crashes/xamarin
-- https://learn.microsoft.com/en-us/appcenter/sdk/distribute/xamarin

- Change (* - For UWP, HoloLens support hasn't been verified.) to (\* - For UWP...) for literal render of (*).

mahab339 committed:
Broken url fix - update to App Center bug tracker integration article

- Change ([work item types API](/rest/api/vsts/wit/work%20item%20types) for additional information.) to (... vsts/wit/) for additional...) as former one displays 404 page.

mahab339 committed:
Typo fix - update App Center webhooks article

- Change (For Distribute, the webhook will be triggered when you when a new version is released to a distribution group.) to (... will be triggered when a new version is released...).

mahab339 committed:
URL update - update to publish an iOS package to the App Store (Production) article

In publish an iOS package to the App Store (Production) article, change (https://developer.apple.com/support/app-review/) to (https://developer.apple.com/app-store/review/). This fixes Apple review article broken URL.

mahab339 committed:
Typo fix - update to Intune company portal distribution article

In Intune company portal distribution article, change (... global admin of your company's Azure Active Directory global admin will need to go to... ) to (... Azure Active Directory will need to go to... ).

lucen-ms commented on docs/general/platform-service-matrix.md:
```diff
@@ -31,6 +31,8 @@ The table below lists all the Operating System and Platforms and their supported
 | Xamarin - iOS          |  ✅   |  ✅  |     ✅       |   ✅       |    ✅     |
 | Xamarin - Android      |  ✅   |  ✅  |     ✅       |   ✅       |    ✅     |
 | Xamarin - macOS        |  ❌   |  ❌  |     ✅       |   ✅       |    ✅     |
+| .NET MAUI - Android    |  ❌   |  ❌  |     ✅       |   ✅       |    ✅     |
```
We don't want to mention .NET MAUI at this moment, please remove.

mahab339 replied:
Done, please review.

lucen-ms commented on docs/dashboard/bugtracker/index.md:
```diff
@@ -34,7 +34,7 @@ App Center has bug tracker integration for the crashes service. Users can be qui
       2. Select which Azure DevOps projects to integrate the bug tracker with and click **Next**.
       3. Toggle **On** for **Auto create ticket**, and fill in the fields for **Area** and **Default Payload**, and click **Add**:
-         - Default payload is an optional field to fill in for use in work items. For example, `{"System.IterationPath": "Area\\Iteration 1", "System.AssignedTo": "Fabrikam"}`. Refer to the [work item types API](/rest/api/vsts/wit/work%20item%20types) for additional information.
+         - Default payload is an optional field to fill in for use in work items. For example, `{"System.IterationPath": "Area\\Iteration 1", "System.AssignedTo": "Fabrikam"}`. Refer to the [work item types API](/rest/api/vsts/wit/) for additional information.
```
The url should be `/rest/api/azure/devops/wit/work-item-types`

mahab339 replied:
Changed, please review.

mahab339 committed:
Minor fixes - changes fix

Address @lucen-ms comments.

mahab339 committed:
Typo fix - update to deploy Azure DevOps builds with App Center article

In deploy Azure DevOps builds with App Center article, change (... for querying for the Store IDs using .) and (... to create a new connection.Paste the API key you copied...) to (... for querying for the Store IDs.) and (... to create a new connection. Paste...) respectively.

mahab339 committed:
Merge branch 'MicrosoftDocs:live' into patch-1

lucen-ms commented:
@mahab339 thank you for your contribution!
